src/desktop_develop.ts

```diff
@@ -411,7 +411,7 @@ export default class DesktopDevelopBuilder {
             const squirrelDir = 'squirrel-windows' + (target.arch === 'ia32' ? '-ia32' : '');
             const archDir = target.arch;
 
-            await fsProm.mkdir(path.join(this.appPubDir, 'install', 'win32', archDir), { recursive: true });
+            await fsProm.mkdir(path.join(this.appPubDir, 'install', 'win32', archDir, 'msi'), { recursive: true });
```

Comment:
Is there a reason for `msi` to have its own directory, while e.g. squirrel does not?